### PR TITLE
client-server connection

### PR DESCRIPTION
**Pull request by elimoreh**

### CODE REVIEW

File: Module/client/src/components/app.jsx [hunk 1]
```diff
@@ -1,15 +1,32 @@
 import React from 'react';
+import axios from 'axios';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      // refers to the id number typed in the url
+      id: window.location.href.match(/id\s*=\s*(.*)/)[1] - 1,
```
Comment (michelledhuang23) [inline Module/client/src/components/app.jsx]: What happens if your state.id is null? If there is no match? (i.e. id=459). Might want to handle invalid inputs.

File: Module/client/src/components/app.jsx [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+    };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    axios.get('/listinginfo').then((response) => {
```
Comment (michelledhuang23) [inline Module/client/src/components/app.jsx]: oooo interesting....will check out axios! I used fetch but it's good to check out others too (:

**Review by michelledhuang23 — APPROVED** [review]
Overall really clean! Please checkout my comment about your this.state.id but other than that you're good to go!

File: Module/client/src/components/app.jsx (hunk 1 above)
Comment (elimoreh) [reply Module/client/src/components/app.jsx]: It now automatically sets the id to one if the input is invalid.

**Review by Lightining1boy — APPROVED** [review]
This all looks pretty good eli keep on going!

**Pull request merged**